### PR TITLE
fix: fix compile error on arm platform

### PR DESCRIPTION
**Pull request by archerny**
replaced the platform dependent crate fasthash.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -39,6 +38,7 @@ spin = "0.5.0"
 tokio-codec = "0.1"
 uuid = "0.7"
 zstd = "0.11"
+murmur2 = "0.1.0"
```
Comment (chunshao90) [inline Cargo.toml:42]: dep order

File: src/location/ob_part_desc.rs [hunk 2]
```diff
@@ -723,7 +723,7 @@ impl ObKeyPartDesc {
 
     // TODO: check if murmur2 hash value is correct(java sdk)
     pub fn long_hash(value: i64, hash_code: i64) -> Result<i64> {
```
Comment (chunshao90) [inline src/location/ob_part_desc.rs:725]: add long_hash test

**Comment by chunshao90** [comment]
close #3

File: src/location/ob_part_desc.rs (hunk 2 above)
Comment (jiacai2050) [reply src/location/ob_part_desc.rs:725]: long_hash is so simple that test is not needed here...

**Review by jiacai2050 — APPROVED** [review]
LGTM

**Pull request merged**